useMethod: 1

abs_position_std: 0.1
gps_position_std: 0.01

imu:
  gyroscope_noise_density: 1.0e-2      # [ rad / s / sqrt(Hz) ]   ( gyro "white noise" )
  gyroscope_random_walk: 1.0e-3        # [ rad / s^2 / sqrt(Hz) ] ( gyro bias diffusion )
  accelerometer_noise_density: 1.0e-1  # [ m / s^2 / sqrt(Hz) ]   ( accel "white noise" )
  accelerometer_random_walk: 1.0e-2    # [ m / s^3 / sqrt(Hz) ].  ( accel bias diffusion )

R_BtoI: [-0.16814, 0.985721, -0.00909486, 0.00628673, -0.00815375, -0.999947, -0.985743, -0.168189, -0.00482599]

# UKF tuning parameters
UKF:
    alpha: 0.4 
    beta:  2.0
    kappa: 0.0
